tests/system/test_active_response/test_netsh_windows_command/test_firewall_alerts/test_firewall_alerts.py

```diff
@@ -0,0 +1,75 @@
+import os
+import pytest
+from tempfile import gettempdir
+
+import wazuh_testing as fw
+from wazuh_testing.tools import configuration as config
+from wazuh_testing import event_monitor as evm
+
+
+# Test cases data
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_cases_file_path = os.path.join(test_data_path, 'test_cases', 'cases_firewall_alerts.yaml')
+alerts_json = os.path.join(gettempdir(), 'alerts.json')
+
+# Playbooks
+configuration_playbooks = ['configuration.yaml']
+events_playbooks = ['generate_events.yaml']
+teardown_playbooks = ['teardown.yaml']
+
+# Configuration
+configurations, configuration_metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
+
+
+def check_event(expected_log):
+    """Check for the espected log.
+
+    Args:
+        expected_log (str): Text to find in the alerts.json file
+    """
+    evm.check_event(callback=expected_log, file_to_monitor=alerts_json, timeout=fw.T_5,
+                    error_message=f"Could not find the event '{expected_log}' in alerts.json file")
+
+
+@pytest.mark.parametrize('metadata', configuration_metadata, ids=cases_ids)
+def test_firewall_alerts(configure_environment, metadata, generate_events):
+    '''
+    description: Check that an alert is generated when the firewall is disabled.
+
+    test_phases:
+        - Set a custom Wazuh configuration.
```

Comment:
Complete this according to the documentation structure

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3787/commits/e4ded61433bf9f440dd1336be139e4ecb01581ec)